e2e-tests/fixtures.ts

```diff
@@ -1,18 +1,31 @@
+import fs from 'fs'
+import path from 'path'
 import { test as base } from '@playwright/test';
 import { ElectronApplication, Page, BrowserContext, _electron as electron } from 'playwright'
+import { randomString } from './utils';
 
 let electronApp: ElectronApplication
 let context: BrowserContext
 let page: Page
+let repoName = randomString(10)
+export let graphDir = path.resolve(__dirname, '../tmp/e2e-graph', repoName)
 
 base.beforeAll(async () => {
   if (electronApp) {
     return ;
   }
 
+  fs.mkdirSync(graphDir, {
+    recursive: true,
+  });
+
   electronApp = await electron.launch({
     cwd: "./static",
     args: ["electron.js"],
+    env: {
+      ...process.env,
```

Comment:
Playwright does not merge process.env.
See https://github1s.com/microsoft/playwright/blob/HEAD/packages/playwright-core/src/client/electron.ts#L52-L53